test/testcafe/spec/DeviceEnrollmentTerminalView_spec.js

```diff
@@ -139,4 +139,145 @@ test
     await t.expect(content).contains('Follow the instructions in your browser to set up MobileIron, then try accessing this app again');
     await t.expect(deviceEnrollmentTerminalPage.getCopyButtonLabel()).eql('Copy link to clipboard');
     await t.expect(deviceEnrollmentTerminalPage.getCopiedValue()).eql('https://anotherSampleEnrollmentlink.com');
-  });
\ No newline at end of file
+  });
+
+test
+  .requestHooks(logger)('shows the correct content in Android ODA terminal view', async t => {
```

Comment:
nit: it might be better to describe it as testing the logic of clicking the button to move back and forth between different pages.